source/type/stop_point.h

```diff
@@ -56,6 +57,7 @@ struct StopPoint : public Header, Nameable, hasProperties, HasMessages {
     std::vector<StopPointConnection*> stop_point_connection_list;
     std::set<Dataset*> dataset_list;
     boost::container::flat_set<Route*> route_list;
```

Comment:
does it become useless?
Because of https://github.com/CanalTP/navitia/pull/3401/files#diff-e0441f873b68f4bee05b0865750b7d996e6e4f706b2d9a297dbacc64dfb9a388R60